azure-iot-device/azure/iot/device/iothub/sync_clients.py

```diff
@@ -454,6 +460,10 @@ def send_message_to_output(self, message, output_name):
         """
         if not isinstance(message, Message):
             message = Message(message)
+
+        if message.get_size() > device_constant.TELEMETRY_MESSAGE_SIZE_LIMIT:
+            raise ValueError("Size of telemetry message can not exceed 256 KB.")
```

Comment:
>raise ValueError [](start = 12, length = 16)

Don't forget to add :raises: for this to the docstring above.  I would also remove the word "telemetry" from the exception string ("Size of message can not exceed 256 KB"